sources/python.py

```diff
@@ -0,0 +1,48 @@
+import random
+import time
+import argparse
+
+def prng(numbers, trials):
+	frequency = [0] * numbers;
+	probability = [0.0] * numbers;
+
+
+	start_time = time.time()
+	
+	for x in range(0, trials):
+		r_number = random.randint(0, numbers-1);
+		frequency[r_number] += 1;
+
+	output = "";
+	for y in range(0, numbers):
+		if(frequency[y] == 0):
+			output += f'{y} : 0\n';
+		else:
+			probability[y] = frequency[y] / float (trials);
+			output += f'{y} : {probability[y]}\n';
+
+
+	duration = time.time() - start_time
+
+	filename = f'python_{numbers}_{trials}.txt';
+
+	with open(filename, 'w') as f:
+		f.write(output);
+
+	print("--- %s seconds ---" % duration);
```

Comment:
This is a good idea however I'd like to limit scope creep -- for now the focus is gathering randomness data from programming languages.

Once we complete the first iteration and properly model some statistics -- V2 we can add more metrics (ie, computation time).  We'll hold off on requiring timing, but I'll allow this code as-is for now.